### PR TITLE
add PR template

### PR DESCRIPTION
**Pull request by zramsay**
and move CONTRIBUTING out of root into `.github`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,15 +0,0 @@
-# Contributing
-Hello fellow contributor! This document will provide an overview of the process for contributing to the CBC casper codebase! Please note that these guidelines may be changed in the future. 
-
-# Getting Started
-To get started with the codebase, see the download and installation instructions in the readme. 
-
-# Making a Pull Request
-Once you have implemented your proposed changes, you need to create a pull request. Before making a PR, make sure there are no merge conflicts with your branch and that all tests pass.
-
-TODO: Add notes about licensing and rights.
```
Comment (zramsay) [inline CONTRIBUTING.md]: this is sort of implied by having the license, no?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,15 +0,0 @@
...
-
-# Making a Pull Request
-Once you have implemented your proposed changes, you need to create a pull request. Before making a PR, make sure there are no merge conflicts with your branch and that all tests pass.
-
-TODO: Add notes about licensing and rights. 
-
-# Resources
-Consensus protocols are notorious for being incredibly hard to understand. Luckily, CBC Casper's spec is much easier to consume than most other consensus protocols that exist today (a result of the correct by construction process). That being said, here are some resources that may be helpful in getting started understanding the protocol. 
-
-TODO: Add Casper papers, add Vlad's talk.
```
Comment (zramsay) [inline CONTRIBUTING.md]: pointed to the wiki instead

File: CONTRIBUTING.md (hunk 1 above)
Comment (djrtwo) [reply CONTRIBUTING.md]: yes

**Comment by djrtwo** [comment]
Looks good. thanks!

**Pull request merged**